src/wallet/crypter.h

```diff
@@ -162,28 +162,26 @@ class CCryptoKeyStore : public CBasicKeyStore
     {
         {
```

Comment:
@benma It certainly looks like it. The lock goes out of scope either way, and that last return is unreachable as it stands now